main.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'app'
+
+def option_list(option, app)
+  options = {
+    1 => :list_all_books,
+    2 => :list_all_people,
+    3 => :create_person,
+    4 => :create_book,
+    5 => :create_rental,
+    6 => :list_rentals_by_id,
+    7 => :exit
+  }
+
+  if options.key?(option)
+    app.send(options[option])
+  else
+    puts 'Invalid option, please try again.'
+  end
+end
+
+def main
+  app = App.new
+  puts 'Welcome to my School Library'
+  loop do
+    puts 'Please choose an option:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person id'
+    puts '7. Exit'
+
```

Comment:
- [ ] [OPTIONAL]
Let's introduce the following improvements to improve interactivity of the console app:
  * the list of people is currently having the same numbering. Let's ensure they are numbered orderly.
  * similarly, all created rentals are numbered _'1'_. Let's ensure order in the numbering here as well:

**Improper numbering of lists:**

![image](https://github.com/UbdaNam/OOP-school-library/assets/12745474/941535b1-576f-4620-b3da-7579230d1c97)

Reply:
Thank you for your suggestion @rloterh, but it shows the same numbering for different types of users. If you add another student or another teacher you will see the order numbering correctly. For the rental, if you add additional rental it will be added as 2nd, 3rd, and soon.

Reply:
@UbdaNam 
Thanks for making thing clearer. However, from the screenshot above you can observe that I have 2 different rentals. How about that too?

Reply:
Hi @rloterh, rentals indexing starts from 1 for every person but, I will consider your suggestion